ada_2_fhir-r4/mp/9.3.0/payload/2.0.0-beta.3/mp-MedicationAdministration2.xsl

```diff
@@ -310,16 +312,7 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
 
                 <xsl:variable name="dosage">
                     <!--  afgesproken_hoeveelheid -->
-                    <xsl:for-each select="afgesproken_hoeveelheid">
-                        <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationAdministration2.AgreedAmount">
-                            <valueQuantity>
-                                <xsl:call-template name="_buildMedicationQuantity">
-                                    <xsl:with-param name="adaValue" select="aantal"/>
-                                    <xsl:with-param name="adaUnit" select="eenheid"/>
-                                </xsl:call-template>
-                            </valueQuantity>
-                        </extension>
-                    </xsl:for-each>
+                    <!-- MP-1395 LR: afgesproken_hoeveelheid removed from MedicationAdministration -->
```

Comment:
You might consider to just remove this comment and line 314 as well, completely.